src/utils/host.ts

```diff
@@ -0,0 +1,5 @@
+// Returns the current browsers address in the format of ${PROTOCOL}//${HOST}
+// i.e. https://dev.easi.cms.gov
+export default function getWindowAddress(): string {
+  return `${window.location.protocol}//${window.location.host}`;
```

Comment:
I think you could get away with `window.location.origin` right?

Reply:
Looks like it! I'll update that

Reply:
be0907806